haxepunk/Tween.hx

```diff
@@ -61,21 +69,18 @@ class Tween extends EventDispatcher
 		_type = type == null ? TweenType.Persist : type;
 		_ease = ease;
 		_t = 0;
-		_callback = complete;
-		super();
-
-		if (_callback != null)
-		{
-			addEventListener(TweenEvent.FINISH, _callback);
-		}
 	}
 
+	/** @private Update function for override in subclasses */
+	function _update() {}
```

Comment:
You could name this one update and the other method _update. This one is more user facing (expected to be overridden) while the other only used internally in Scene.update. Might also consider making the other update function a private method and adding an @:allow.

Signal name collisions are going to be a headache - maybe we should use an on-, pre-, post- convention (so onUpdate, onStart here) and convert the other ones in Scene/Engine to match? Whatever we go with we should try to be consistent.